django/db/models/fields/json.py

```diff
@@ -134,8 +134,11 @@ def validate(self, value, model_instance):
                 params={"value": value},
             )
 
+    def to_python(self, value):
+        return json.loads(value)
+
     def value_to_string(self, obj):
-        return self.value_from_object(obj)
+        return json.dumps(self.value_from_object(obj))
```

Comment:
This is a separate issue(?) that I noticed while trying to find a suitable way to setup a unit test: JSONField wasn't doing any proper json conversion to/from string values.  Perhaps nobody cares 🤷‍♂️